src/simsopt/field/coil.py

```diff
@@ -1,29 +1,45 @@
 from math import pi
 import numpy as np
+from jax import vjp, jacrev
 
 from simsopt._core.optimizable import Optimizable
 from simsopt._core.derivative import Derivative
 from simsopt.geo.curvexyzfourier import CurveXYZFourier
 from simsopt.geo.curve import RotatedCurve
+from simsopt.geo.jit import jit
 import simsoptpp as sopp
+from simsopt.field.force import regularization_circ
 
-
-__all__ = ['Coil', 'Current', 'coils_via_symmetries', 'load_coils_from_makegrid_file',
+__all__ = ['Coil', 'JaxCurrent',
+           'Current', 'coils_via_symmetries',
+           'load_coils_from_makegrid_file',
            'apply_symmetries_to_currents', 'apply_symmetries_to_curves',
-           'coils_to_makegrid', 'coils_to_focus'
+           'coils_to_makegrid', 'coils_to_focus', 'coils_to_vtk'
            ]
 
 
 class Coil(sopp.Coil, Optimizable):
     """
-    A :obj:`Coil` combines a :obj:`~simsopt.geo.curve.Curve` and a
-    :obj:`Current` and is used as input for a
-    :obj:`~simsopt.field.biotsavart.BiotSavart` field.
+    Represents a magnetic coil as a combination of a geometric curve and an electric current.
+
+    This class combines a :class:`~simsopt.geo.curve.Curve` and a :class:`Current` object, and is used as input for
+    :class:`~simsopt.field.biotsavart.BiotSavart` field calculations. 
+
+    Parameters
+    ----------
+    curve : simsopt.geo.curve.Curve
+        The geometric curve describing the coil shape.
+    current : Current
+        The current object describing the electric current in the coil.
+    regularization : Regularization
+        The regularization object for the coil corresponding to the coil cross section. 
+        Default is a circular cross section with radius 0.05.
     """
 
-    def __init__(self, curve, current):
+    def __init__(self, curve, current, regularization=regularization_circ(0.05)):
```

Comment:
I'm not sure about by default making a coil have a regularization, when some problems don't have this.  I'd lean towards making this be `None`

Reply:
a second comment about this is adding a new attribute to `Coil` might break a lot of users' jsons... a suggestion would be create a wrapper coil class like `RegularizedCoil` but I'm open to other solutions that require less code.